src/compile/mark/mark.ts

```diff
@@ -36,7 +38,7 @@ export function parseMark(model: UnitModel): any[] {
 function parsePathMark(model: UnitModel) { // TODO: extract this into compilePathMark
   const mark = model.mark();
   // TODO: replace this with more general case for composition
-  const hasParentData = !!model.parent();
+  const isFaceted = model.parent() && model.parent().isFacet();
```

Comment:
Will do in follow up

On Sat, Mar 19, 2016 at 4:31 PM Kanit Wongsuphasawat <
notifications@github.com> wrote:

> In src/compile/mark/mark.ts
> <https://github.com/vega/vega-lite/pull/1241#discussion_r56756809>:
>
> > @@ -36,7 +38,7 @@ export function parseMark(model: UnitModel): any[] {
> >  function parsePathMark(model: UnitModel) { // TODO: extract this into compilePathMark
> >    const mark = model.mark();
> >    // TODO: replace this with more general case for composition
> > -  const hasParentData = !!model.parent();
> > +  const isFaceted = model.parent() && model.parent().isFacet();
>
> Sure, we should at least put the right name in and add TODO for the splom
> and grand parent case.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/vega/vega-lite/pull/1241/files/2d40fd567bcc48dcc87430cf20a363eb191f8d32#r56756809>
>